### PR TITLE
Add a tutorial on SSH usage

### PR DESCRIPTION
**Pull request by Waterdrips**
Signed-off-by: Alistair Hey <alistair@heyal.co.uk>

Example for SSH use

### CODE REVIEW

File: docs/ssh-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+First of all, download `inletsctl` to your local machine
+```sh
+# Install Inletsctl to /usr/local/bin
+curl -sLSf https://inletsctl.inlets.dev | sudo sh
+
+# Create an exit node
+
+inletsctl create --access-token-file ~/do-access-token --remote-tcp 127.0.0.1
+```
+This should then create an exit node, which will forward all TCP traffic to through the client to the `remote-tcp` address
```
Comment (alexellis) [inline docs/ssh-tutorial.md]: Did this work? I seemed to need the IP of eth0

File: docs/ssh-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+sudo inlets-pro server \
+  --auto-tls \
+  --common-name $IP \
+  --remote-tcp 127.0.0.1 \
+  --token $TOKEN
+```
+
+
+## Configure your ssh agent
+(port 22 conflict with exit-node)
```
Comment (alexellis) [inline docs/ssh-tutorial.md]: This doesn’t seem like a complete sentence

File: docs/ssh-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+If we find the line in  `/etc/ssh/sshd_config` near the top that is `Port 22`, make sure it's un-commented
+and add `Port 2222`
+
+```
+Port 22
+Port 2222
+```
+
+Next we recommend disabling password authentication, and instead relying on SSH Keys only.
```
Comment (alexellis) [inline docs/ssh-tutorial.md]: Replace we with it is recommended

File: docs/ssh-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  export LICENSE=""
+  inlets-pro client --connect "wss://206.189.114.179:8123/connect" \
+        --token "4NXIRZeqsiYdbZPuFeVYLLlYTpzY7ilqSdqhA0HjDld1QjG8wgfKk04JwX4i6c6F" \
+        --license "$LICENSE" \
+        --tcp-ports $TCP_PORTS
+
+```
+
+
+#### Manually
```
Comment (alexellis) [inline docs/ssh-tutorial.md]: Manual setup of the exit node

File: docs/ssh-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+and add `Port 2222`
+
+```
+Port 22
+Port 2222
+```
+
+Next we recommend disabling password authentication, and instead relying on SSH Keys only.
+
+> WARNING: make sure you have validated that SSH works using your SSH Key before disabling password authentication.
```
Comment (alexellis) [inline docs/ssh-tutorial.md]: Maybe suggest ssh-copy-id

File: docs/ssh-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
+# Tutorial
+
+Create a reverse proxy to allow SSH access to a computer
+
+Scenario: You want to allow SSH access to a computer that doesn't have a public IP, is inside a private network or behind
+a firewall
```
Comment (alexellis) [inline docs/ssh-tutorial.md]: Mention remote access to a raspberry pi as a usecase for SEO

File: docs/ssh-tutorial.md (hunk 5 above)
Comment (Waterdrips) [reply docs/ssh-tutorial.md]: Wow i have been doing this manually for years.

File: docs/ssh-tutorial.md (hunk 1 above)
Comment (Waterdrips) [reply docs/ssh-tutorial.md]: You are correct :man_facepalming: wrote this in 2 halves and must have not tested the full flow.

Changed and tested full flow now

**Pull request merged**